lib/registrant.rb

```diff
@@ -0,0 +1,35 @@
+class Registrant
+  attr_reader :name, :age, :permit, :license_data
```

Comment:
Careful here, only make attr_readers for the methods you want to expose. We don't need the method `#permit` since we have `#permit?`